src/models/train_model.py

```diff
@@ -132,10 +132,12 @@ def predict(trained_model, data_model, best_model_features,
     preds = trained_model.predict_proba(data_model[features])[::, 1]
     df_pred = data_model.copy(deep=True)
     df_pred['prediction'] = preds
-    if target=='crash':
+    if target == 'crash':
         fn = 'seg_with_predicted'
     else:
         fn = 'seg_with_predicted_%s' % target
+        df_pred = df_pred.rename(columns={target: 'crash'})
```

Comment:
Updated! I also added a brief comment to the README describing how the showcase would look, but I had previously added information on how to add the split columns to the config, so that might be enough.